### ISSUE TITLE
Finetuning hyperparameter w

### ISSUE THREAD

**Issue by abc-125** [arimousa/DDAD#17]
Hello, thank you for your excellent work! I wanted to ask how you got values for hyperparameter `w` for MVTec AD  and VisA. I was not able to find it in the paper, sorry if I have missed something.

**Comment by arimousa** [comment]
Hello, appreciate your inquiry. 

Setting w is based on hyperparameters optimisation. We have not proposed a solid solution to set it automatically and it can be an interesting avenue for future works.

I hope this explanation adequately addresses your question.

**Comment by abc-125** [comment]
Thank you for your reply, what data do you use for the optimization of `w`?

**Comment by arimousa** [comment]
If no validation set is available for hyperparameter tuning, a prudent option is to employ synthetic anomalies. Some papers, such as DRAEM, utilize DTD as synthetic data during training. The same synthetic anomalies can be adopted to identify the optimal configuration in the absence of a validation set.

DRAEM: https://arxiv.org/abs/2108.07610
DTD: https://www.robots.ox.ac.uk/~vgg/data/dtd/
